evm_rpc_types/src/lib.rs

```diff
@@ -165,6 +166,45 @@ macro_rules! impl_hex_string {
     };
 }
 
+impl_hex_string!(HexByte(Byte));
 impl_hex_string!(Hex20([u8; 20]));
 impl_hex_string!(Hex32([u8; 32]));
+impl_hex_string!(Hex256([u8; 256]));
 impl_hex_string!(Hex(Vec<u8>));
+
+/// A wrapper to be able to decode single character hex string
+/// such as `0x0` or `0x1` into a byte. By default,
```

Comment:
So, `0x1` is interpreted as `0x01`, right?
Do odd lengths greater than 1 still return an error or are they patched too?

Reply:
Here is a [documentation page](https://pkg.go.dev/github.com/ethereum/go-ethereum/common/hexutil) which explains the logic for integer hex values:

> Integers are encoded using the least amount of digits (no leading zero digits). Their encoding may be of uneven length. The number zero encodes as "0x0".

Reply:
@THLO 
> So, 0x1 is interpreted as 0x01, right?

Yes, see the test `should_decode_single_hex_char_into_hex_byte`

> Do odd lengths greater than 1 still return an error or are they patched too?

They return an error: Any length greater than 1 is passed as is to `hex::FromHex::from_hex`, which returns an error in case of odd length.